### DISCUSSION TITLE
xwd status

### DISCUSSION THREAD

**Discussion by WurmD** [Homebrew/discussions#4018] [Casks]
xwd does not seem to have ever been installable by brew, but `webrick` and `rack` seem to use it? https://github.com/search?q=org%3AHomebrew+xwd&type=code

`port` [is able to install it](https://ports.macports.org/port/xwd/), is there any inkling if`xwd` will be added to brew?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew documents how to do that. The search only shows a mimetype map, it doesn't show any dependencies.
